client/packages/system/src/Item/DetailView/Tabs/ItemVariants/ItemVariantsTab.tsx

```diff
@@ -16,134 +14,81 @@ import {
-                id: Date.now().toString(),
-                name: '',
-                packagingVariants: [...dummyPackVariants],
-              },
-            ]);
-          }}
+          onClick={() => onOpen()}
           label={t('label.add-variant')}
         />
       </AppBarButtonsPortal>
       <Box flex={1} marginX={2}>
-        {variants.map((v, idx) => (
-          <ItemVariant variant={v} setVariant={setVariant} index={idx} />
+        {itemVariants.map(v => (
+          <ItemVariant key={v.id} variant={v} onOpen={onOpen} />
         ))}
       </Box>
-      <AppFooterPortal
-        Content={
-          <Box
-            gap={2}
-            display="flex"
-            flexDirection="row"
-            justifyContent="flex-end"
-            alignItems="center"
-            height={64}
-          >
-            <DialogButton onClick={() => {}} variant="cancel" />
-            <DialogButton onClick={() => {}} variant="save" />
-          </Box>
-        }
-      />
     </>
   );
 };
 
 const ItemVariant = ({
   variant,
-  index,
-  setVariant,
+  onOpen,
 }: {
   variant: ItemVariantFragment;
-  index: number;
-  setVariant: (variant: ItemVariantFragment) => void;
+  onOpen: (variant?: ItemVariantFragment) => void;
 }) => {
   const t = useTranslation();
 
   return (
-    <Box maxWidth="1000px" margin="25px auto">
-      <Typography variant="h5" fontWeight="bold">
-        {variant.name || `${t('label.variant')} ${index + 1}`}
-      </Typography>
+    <Box maxWidth="1000px" margin="25px auto 75px">
+      <Box display="flex" justifyContent="space-between" alignItems="end">
+        <Typography variant="h6" fontWeight="bold" color="black">
+          {variant.name}
+        </Typography>
+        <Box display="flex" gap={2}>
+          <FlatButton
+            label={t('label.edit')}
+            onClick={() => onOpen(variant)}
+            startIcon={<EditIcon />}
+            color="primary"
+          />
+          <FlatButton
+            label={t('label.delete')}
+            onClick={() => {}}
+            startIcon={<DeleteIcon />}
+            color="primary"
+          />
+        </Box>
+      </Box>
 
-      <Box justifyContent="center" display="flex" gap={2}>
-        <Box
-          marginTop={5}
-          display="flex"
-          flexDirection="column"
-          gap={1}
-          flex={1}
-        >
+      <Box justifyContent="center" display="flex" gap={2} alignItems={'center'}>
+        <Box display="flex" flexDirection="column" gap={1} flex={1}>
           <InputWithLabelRow
             label={t('label.name')}
             labelWidth="200"
-            Input={
-              <BasicTextInput
-                value={variant.name}
-                onChange={event => {
-                  setVariant({ ...variant, name: event.target.value });
-                }}
-                fullWidth
-              />
-            }
+            Input={<BasicTextInput value={variant.name} disabled fullWidth />}
```

Comment:
Could have reused the edit form from the modal here, but am under the impression that item variants are likely to end up with a lot of fields, and probably only want a subset in the tab view 👀

Reply:
Really? I would have thought we'd want to see the same fields on the tab view as in the modal. How many are we talking?

Reply:
I think we're talking long term, but that's what Adam has said 😁 I think I'll stick with it anyway - nice not to do things like query all manufacturers for the dropdown, only to display read only data